doc/7-systems/README.md

```diff
@@ -1,56 +1,813 @@
+
+- **Sega Genesis**: [YM3438](ym2612.md), [SN76489](sms.md)
+- **Sega Genesis (with Sega CD)**: [YM3438](ym2612.md), [SN76489](sms.md), [RF5C164](ricoh.md)
+<!-- -->
+- **Sega Master System**: [SN76489](sms.md)
+- **Sega Master System (with FM expansion)**: [SN76489](sms.md), [YM2413](opll.md)
+<!-- -->
+- **NES**: [2A03](nes.md)
+- **Famicom with Konami VRC6**: [2A03](nes.md), [VRC6](vrc6.md)
+- **Famicom with Konami VRC7**: [2A03](nes.md), [VRC7](opl.md)
+- **Famicom with MMC5**: [2A03](nes.md), [MMC5](mmc5.md)
+- **Famicom with Sunsoft 5B**: [2A03](nes.md), [5B](ay8910.md)
+- **Famicom with Namco 163**: [2A03](nes.md), [N163](n163.md)
+- **Famicom Disk System**: [2A03](nes.md), [FDS](fds.md)
+<!-- -->
+- **Game Boy**: [Game Boy](game-boy.md)
+<!-- -->
+- **SNES**: [SNES](snes.md)
+<!-- -->
+- **NEC PC Engine/TurboGrafx-16**: [HuC6280](pce.md)
+<!-- -->
+- **Commodore VIC-20**: [VIC](vic20.md)
+<!-- -->
+- **Commodore 64 (6581 SID)**: [MOS 6581](c64.md)
+- **Commodore 64 (8580 SID)**: [MOS 8580](c64.md)
+<!-- -->
+- **Amiga**: [Amiga](amiga.md)
+<!-- -->
+- **Arcade (YM2151 and SegaPCM)**: [YM2151](ym2152.md), [SegaPCM](segapcm.md)
+<!-- -->
+- **Capcom CPS-1**: [YM2151](ym2152.md), [MSM6295](msm6295.md)
+- **Capcom CPS-2 (QSound)**: [QSound](qsound.md)
+<!-- -->
+- **Neo Geo CD**: [YM2610](ym2610.md)
+- **Neo Geo CD (extended channel 2)**: [YM2610](ym2610.md)
+<!-- -->
+- **Neo Geo Pocket**: [T6W28](t6w28.md), [DAC](dac.md)
+<!-- -->
+- **Atari 2600/7800**: [TIA](tia.md)
+- **Atari 800**: [POKEY](pokey.md)
+<!-- -->
+- **Konami Bubble System**: [AY-3-8910](ay8910.md) × 2, [Konami WSG](bubblesystem.md)
+<!-- -->
+- **Sharp X68000**: [YM2151](ym2152.md), [MSM6258](msm6258.md)
+<!-- -->
+- **PC + Sound Blaster Pro**: [YM3812](opl.md) × 2, [DAC](dac.md), [PC Speaker](pcspkr.md)
+<!-- -->
+- **MSX**: [AY-3-8910](ay8910.md)
+- **MSX + SCC**: [YM2149(F)](ay8910.md), [SCC](scc.md)
+<!-- -->
+- **Commander X16 (VERA only)**: [VERA](vera.md)
+<!-- -->
+- and many, many more!
+
+
+
+# chips
+
+this is the full list of chips that Furnace supports.
+
+- [2A03](nes.md)
 - [Amiga](amiga.md)
-- [AY-3-8910](ay8910.md)
-- [Microchip AY8930](ay8930.md)
-- [Bubble System WSG](bubblesystem.md)
-- [Casio PV-1000](pv1000.md)
-- [Commodore 64](c64.md)
-- [Commodore PET](pet.md)
-- [Commodore VIC-20](vic20.md)
-- [Generic PCM DAC](dac.md)
-- [Famicom Disk System](fds.md)
+- [AY-3-8910, AY-3-8914, YM2149(F), 5B](ay8910.md)
+- [AY8930](ay8930.md)
+- [MOS 6581, MOS 8580](c64.md)
+- [ES5506](es5506.md)
+- [SCC](scc.md)
+- [FDS](fds.md)
 - [Game Boy](game-boy.md)
-- [Konami K007232](k007232.md)
-- [Konami K056320](k056320.md)
-- [Konami SCC](scc.md)
-- [Konami VRC6](vrc6.md)
-- [Atari Lynx](lynx.md)
-- [Namco 163](n163.md)
-- [Namco WSG](namco.md)
-- [NES](nes.md)
-- [Nintendo MMC5](mmc5.md)
-- [OKI MSM5232](msm5232.md)
-- [OKI MSM6258](msm6258.md)
-- [OKI MSM6295](msm6295.md)
-- [PC Engine/TurboGrafx-16](pce.md)
+- [Generic PCM [DAC](dac.md)](dac.md)
```

Comment:
Any reason for this nested link?

Reply:
Runaway regex. I thought I caught this one!

Reply:
It happens.